modules/ocf_mirrors/manifests/tails.pp

```diff
@@ -9,6 +9,12 @@
       recurse => true;
   }
 
+  ocf_mirrors::monitoring { 'tails':
+    type          => 'ts',
+    upstream_host => 'mirrors.kernel.org',
+    ts_path       => 'project/trace',
```

Comment:
Instead of relying on another mirror here (albeit a very reliable one), I'd use the upstream at https://archive.torproject.org/amnesia.boum.org/tails/project/trace

(It was pretty hard to find the upstream here, but given the tails mirroring instructions say to mirror from rsync.torproject.org which goes to the same DNS entry as archive.torproject.org, this should be the true upstream)

Reply:
omg thank you for finding this. i was looking forever yesterday and just gave up